.github/workflows/nomad-demo.yaml

```diff
@@ -0,0 +1,33 @@
+name: "Run demo performance tests on Nomad cluster"
+
+on:
+  workflow_dispatch:
+    inputs:
+      holochain-bin-url:
+        description: "The URL to download the `holochain` binary from"
+        required: false
+      commit-hash:
+        description: "The commit hash to checkout before running the tests"
+        required: false
+  pull_request:
+    paths:
+      - ".github/workflows/nomad*.yaml"
+      - ".github/actions/nomad/action.yaml"
```

Comment:
We run demo runs on PRs or manual trigger. The user can specify a custom holochain binaries, and/or a custom commit hash.